build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    kotlin("jvm") version "1.8.22" apply false
+    id("com.google.protobuf") version "0.9.3" apply false
+    id("com.exactpro.th2.gradle.grpc") version "0.0.3" apply false
+    id("com.exactpro.th2.gradle.base") version "0.0.3"
```

Comment:
suggestion: I think it would be better if we put plugins we apply first and then all other plugins